export function handleError(error) {
  console.log(`Error happened: ${error}`);
}
